### PR TITLE
Add wrapper on `creator.addAlias`.

### PR DESCRIPTION
**Pull request by mgautierfr**
Need https://github.com/openzim/libzim/pull/833
Blocked by https://github.com/openzim/libzim/issues/837

### CODE REVIEW

File: tests/test_libzim_creator.py [hunk 1]
```diff
@@ -467,6 +467,28 @@ def test_creator_redirection(fpath, lipsum_item):
...
+    c = Creator(fpath)
+    with pytest.raises(RuntimeError, match="not started"):
+        c.add_redirection("home", "hello", HOME_PATH, {Hint.FRONT_ARTICLE: True})
+    del c
+
+    with Creator(fpath) as c:
+        c.add_item(lipsum_item)
+        c.add_clone("home", "hello", HOME_PATH, {Hint.FRONT_ARTICLE: True})
+        with pytest.raises(RuntimeError, match="doesn't exist"):
+            c.add_clone("accueil", "bonjour", HOME_PATH+"_no_existitant", {Hint.FRONT_ARTICLE: True})
```
[the remark this answers is not in the document]
Comment (mgautierfr) [reply tests/test_libzim_creator.py]: Well, this way we are sure it is non existent :)

Fixed.

**Comment by mgautierfr** [comment]
Adapted to add_alias.

**Review by rgaudin — APPROVED** [review]
👍  Turning this into a WIP so we don't merged until there's at least a nightly we can validate all targets against

**Comment by kelson42** [comment]
This PR is blocked by https://github.com/openzim/libzim/issues/837 (written in description)

**Pull request merged**